src/Components/SignUpPage.js

```diff
@@ -35,8 +45,8 @@ return (
       <Field text="type" name="password" placeholder="Password" />
```

Comment:
Need to change the type on password from 'text' to 'password' that way you get the hidden password text when you type in the input box